tests/cancel_safety.rs

```diff
@@ -0,0 +1,34 @@
+use async_rdma::LocalMrReadAccess;
+use async_rdma::LocalMrWriteAccess;
+use async_rdma::Rdma;
+use std::{alloc::Layout, io, time::Duration};
+mod test_utilities;
+use test_utilities::test_server_client;
+
+async fn client(rdma: Rdma) -> io::Result<()> {
+    const LEN: usize = 600 * 1024 * 1024;
+    let mut lmr = rdma.alloc_local_mr(Layout::new::<[u8; LEN]>())?;
+    let mut rmr = rdma
+        .request_remote_mr(Layout::new::<[u8; LEN]>())
+        .await
+        .unwrap();
+    let _ = tokio::time::timeout(Duration::from_nanos(100), rdma.write(&lmr, &mut rmr)).await;
```

Comment:
How to guarantee 100ns is enough?

Reply:
There's no guarantee, do you have a better idea?

Reply:
One better way is to insert delay manually in the test case, making sure the latency is large enough.

Reply:
insert delay when feature "cancel_safety_test" enabled
eaa7325f785df30c35fc1e16e5ec890dc147c6a5